setup/generate_env_files.py

```diff
@@ -47,7 +47,7 @@ def generate_env_files(keys_path, env_dir):
         "node-address": keys["node-address"]
     }
 
-    with open("config.json", 'w') as f:
+    with open("/home/ubuntu/setup/config.json", 'w') as f:
```

Comment:
```suggestion
    with open("$HOME/setup/config.json", 'w') as f:
```

Suggest you just use `$HOME` to make compatible with other Linux distros